### PR TITLE
Remove mwan and add otb-tracker

### PR DESCRIPTION
**Pull request by Tikiwinkie**

### CODE REVIEW

File: otb-tracker/init [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	validate_section "default"
+	procd_open_instance
+	procd_set_param command /bin/otb-tracker-dns -t "$timeout" -h "$host" -i "$interface" -r "$retry" -d "$domain" -s "$interval"
+	procd_set_param respawn procd_set_param stderr 1
+	procd_set_param stdout 1
+	procd_close_instance
+}
+
+start_service() {
+	otb_tracker_generate_conf default
```
Comment (angt) [inline otb-tracker/init]: just use default values when there is no conf, do not write a useless "default conf"

File: otb-tracker/init [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	config_load network
+	config_foreach _start_track interface
+}
+
+service_triggers() {
+	procd_add_reload_trigger otb-tracker
```
Comment (gregdel) [inline otb-tracker/init]: You can add them all on the same line:
From the [doc](https://wiki.openwrt.org/inbox/procd-init-scripts):
```
service_triggers()
{
        procd_add_reload_trigger "uci-file-name" "second-uci-file"
        procd_add_network_trigger "lan"|"etho0" FIXME - this is still a work in process....
}
```

File: otb-tracker/Makefile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+define Package/$(PKG_NAME)/install
+	$(INSTALL_DIR) $(1)/bin
+	$(INSTALL_BIN) otb-tracker-* $(1)/bin/
+	$(INSTALL_DIR) $(1)/lib
+	$(INSTALL_DATA) lib $(1)/lib/$(PKG_NAME)
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) init $(1)/etc/init.d/$(PKG_NAME)
+	$(INSTALL_DIR) $(1)/usr/share/otb/tracking.d
+	mkdir $(1)/etc/config
```
Comment (gregdel) [inline otb-tracker/Makefile]:
```
$(INSTALL_DIR) $(1)/etc/config
```

File: otb-tracker/Makefile [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Kevin Lanvin <kevin.lanvin@corp.ovh.com>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)
+  SECTION:=OVH
+  CATEGORY:=OverTheBox
+  TITLE:=otb-tracker
```
Comment (angt) [inline otb-tracker/Makefile]: TITLE:=OverTheBox Tracker ?

**Comment by gregdel** [comment]
Fixes #489

**Pull request merged**